base_layer/core/src/base_node/state_machine_service/states/block_sync.rs

```diff
@@ -110,13 +100,14 @@ impl Default for BlockSyncConfig {
     fn default() -> Self {
         Self {
             sync_strategy: BlockSyncStrategy::ViaBestChainMetadata(BestChainMetadataBlockSync),
-            max_metadata_request_retry_attempts: MAX_METADATA_REQUEST_RETRY_ATTEMPTS,
-            max_header_request_retry_attempts: MAX_HEADER_REQUEST_RETRY_ATTEMPTS,
-            max_block_request_retry_attempts: MAX_BLOCK_REQUEST_RETRY_ATTEMPTS,
-            max_add_block_retry_attempts: MAX_ADD_BLOCK_RETRY_ATTEMPTS,
-            header_request_size: HEADER_REQUEST_SIZE,
-            block_request_size: BLOCK_REQUEST_SIZE,
+            max_metadata_request_retry_attempts: 3,
```

Comment:
I don't think this is an improvement. Either leave the constants in or add a config setting for them. Perhaps a `default()` method doesn't make sense and all the values should be provided from config?

Reply:
This default method was out of sorts with how others were done. Default values are generally passed directly, not via single use local constants, to improve readability it seems.
I do think we need the dafault method in this case. Generally the config settings overrided these where applicable.
I will revert to using the local constants here if you still think that is peferrable?